lib/schema/schema.js

```diff
@@ -513,3 +521,7 @@ exports.makeId = function(datum) {
 
   return exports.generateId(datum, idFields);
 };
+
+exports.generateHash = duplicate.generateHash;
```

Comment:
Perhaps include "platform" name in exports so it is clear it is for the platform deduplication.